internal/storage/utilities.go

```diff
@@ -0,0 +1,38 @@
+// +build linux
+
+package storage
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// export this variable so it can be mocked to aid in testing for consuming packages
+var Filepathglob = filepath.Glob

```

Comment:
I'm confused why you are exporting these. The test package as access to this unexported so it doesnt need to be exported to mock.

Reply:
Because I also mock it out for my pci package test :)